karma.conf.js

```diff
@@ -111,14 +111,11 @@ module.exports = function(config) {
     // - config.LOG_INFO
     // - config.LOG_DEBUG
     logLevel: config.LOG_INFO,
-    // enable / disable watching file and executing tests whenever any file changes
-    autoWatch: false,
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
     browsers: process.env.TEST_HEADLESS ? ['ChromeHeadless'] : ['Chrome', 'Firefox'],
     // Continuous Integration mode
     // if true, Karma captures browsers, runs the tests and exits
-    singleRun: true,
```

Comment:
@patrick-tolosa , why was the `singleRun` removed?

Reply:
singleRun in the config will override CLI arguments, for example in our case, we already pass it as an argument when running the command:
`karma start --single-run --browsers=ChromeHeadless`

since this configuration is not requires for watch mode, I just removed it.